### DISCUSSION TITLE
How can I filter words out of the auto-generated slug?

### DISCUSSION THREAD

**Discussion by Gusterson** [strapi/strapi#7981] [Legacy Topics]
I'm loving the auto-generated 'slug' field linked to the title of my posts, but would like to fine-tune the results. Specifically, automatically strip out 'a', 'the', and other "weak" words. Where do I tap into this functionality?

**Comment by lauriejim** [comment]
Hello! I this for this need, you will probably prefer using this guide.
It will not be generated directly in live into the admin panel, but you will be able to manage the JS logic you want to apply.
https://strapi.io/documentation/v3.x/guides/slug.html
